api4/apitestlib.go

```diff
@@ -45,6 +49,7 @@ type TestHelper struct {
 
 	Context              *request.Context
 	Client               *model.Client4
+	GraphQLClient        *model.Client4
```

Comment:
If we are going to use `v5` for the API, would it make sense this be a new Client? I mean would it confuse the users of the library?

Reply:
Yeah, good point. I'll create a new one. :+1:

Reply:
@isacikgoz - I created a separate graphql client in the api4 package itself to prevent it from polluting the model API. I'd want to move it out there when we have a stable API and the methods are more fleshed out.